src/components/tx/modals/NewTxModal/index.tsx

```diff
@@ -68,8 +107,14 @@ const NewTxModal = ({ onClose, recipient }: { onClose: () => void; recipient?: s
               Send NFTs
             </TxButton>
 
+            {txNonce && (
+              <TxButton onClick={onRejectModalOpen} variant="outlined">
+                Empty transaction
+              </TxButton>
+            )}
+
             {/* Contract interaction via Transaction Builder */}
-            {txBuilder.app && !recipient && (
+            {txBuilder.app && !recipient && !txNonce && (
```

Comment:
Same here, this would fail for nonce 0
<img width="605" alt="Screenshot 2022-11-07 at 12 13 09" src="https://user-images.githubusercontent.com/5880855/200296717-a5bcc709-917d-4a38-9b08-0e313b5d1634.png">